cmake/3rdParty/Platform/Android/BuiltInPackages_android.cmake

```diff
@@ -32,3 +31,9 @@ ly_associate_package(PACKAGE_NAME OpenSSL-1.1.1o-rev2-android        TARGETS Ope
 ly_associate_package(PACKAGE_NAME zlib-1.2.11-rev5-android           TARGETS ZLIB            PACKAGE_HASH 73c9e88892c237a3fc6eafc04268ccd9d479e6d55f9df2ed58b236c8f9cf2cae)
 ly_associate_package(PACKAGE_NAME lz4-1.9.3-vcpkg-rev4-android       TARGETS lz4             PACKAGE_HASH f5b22642d218dbbb442cae61e469e5b241c4740acd258c3e8678e60dec61ea93)
 
+set(AZ_USE_PHYSX5 OFF CACHE BOOL "When ON PhysX Gem will use PhysX 5 SDK")
```

Comment:
Since Android uses the script `generate_android_project.py` on top of CMake, currently there is no way to pass `-DAZ_USE_PHYSX5=ON` option to enable PhysX 5 on Android. We have to add an input parameter to `generate_android_project.py` and modify the scripts to be able to use PhysX 5 on Android.

Reply:
There is actually a way to pass input parameters to CMake via `generate_android_project.py`
The [--extra-cmake-configure-args](https://github.com/o3de/o3de/blob/development/cmake/Tools/Platform/Android/generate_android_project.py#L242-L244) argument can be used to supply arguments to the CMake.